### ISSUE TITLE
Fails to start with module error.

### ISSUE THREAD

**Issue by ShaneDelmore** [jacobwgillespie/plex-sync#2]
I have tried to install this on three different computers to make sure it was not something in my environment, two macs and inside the stock node linux dockerhub image (docker pull node) and all three returned the same result when trying to run plex-sync after installing:
```

module.js:442
    throw err;
    ^

Error: Cannot find module '../lib/index'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/plex-sync/bin/plex-sync:2:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

I am using node v6.3.1 and npm v3.10.3

Is there anything I should need to do to get past this error besides the following?
npm install -g plex-sync

**Comment by jacobwgillespie** [comment]
My bad, it was accidentally ignoring the lib folder when publishing.  That's fixed now (in `v0.2.3`).  🎉

**Comment by ShaneDelmore** [comment]
Works great now, thanks for the useful utility.
